packages/admin-ui/src/pages/system/components/Network/HttpsMappings.tsx

```diff
@@ -16,13 +16,60 @@ import newTabProps from "utils/newTabProps";
 import { ReqStatus, HttpsPortalMapping, ExposableServiceInfo } from "types";
 import { httpsPortalDnpName } from "params";
 import "./https-mapping.scss";
+import { ExposableServiceMapping } from "common";
+import Button from "components/Button";
 
 export function HttpsMappings() {
   const [reqStatus, setReqStatus] = useState<ReqStatus>({});
   const mappings = useApi.httpsPortalExposableServicesGet();
   const dnpsRequest = useApi.packagesGet();
   const dappnodeIdentity = useSelector(getDappnodeIdentityClean);
 
+  /** Refresh HTTPs Portal mapping */
+  async function refreshMapping(httpsMapping: ExposableServiceMapping[]) {
+    if (reqStatus.loading) return;
+
+    try {
+      await confirmPromise({
+        title: "Refresh",
+        text: "Refresh HTTPs portal mapping",
+        label: "Refresh",
+        variant: "dappnode"
+      });
+
+      setReqStatus({ loading: true });
+      await withToast(
+        () =>
+          Promise.all(
+            httpsMapping.map(httpsMapping =>
+              api.httpsPortalMappingRemove(httpsMapping)
+            )
+          ),
+        {
+          message: "Removing HTTPs mapping...",
+          onSuccess: "Removed HTTPs mapping"
+        }
+      );
+      await withToast(
+        () =>
+          Promise.all(
+            httpsMapping.map(httpsMapping =>
+              api.httpsPortalMappingAdd(httpsMapping)
+            )
+          ),
+        {
+          message: "Adding HTTPs mapping...",
+          onSuccess: "Added HTTPs mapping"
+        }
+      );
+      setReqStatus({ result: true });
```

Comment:
This strategy is not safe. If the DAPPMANAGER <-> UI communications interrupt the "refresh mappings" operation will break at an unpredictable spot losing information. You should a new route called refreshMappings or something and do this atomically in the DAPPMANAGER side. And do it one be one:
```
for mapping:
  delete mapping
  add mapping
```

instead of 
```
for mapping:
  delete mapping
for mapping:
  add mapping
```